coa/pkg/logger/coa-logger.go

```diff
@@ -7,32 +7,47 @@
 package logger
 
 import (
+	"context"
+	"fmt"
 	"os"
+	"runtime"
+	"sync"
 	"time"
 
+	"github.com/eclipse-symphony/symphony/coa/pkg/logger/hooks"
 	"github.com/sirupsen/logrus"
 )
 
 // daprLogger is the implemention for logrus
 type daprLogger struct {
```

Comment:
This is long overdue but we should rename this to coaLogger

Reply:
Done.